File: week_1_assignment/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = get_int("Height: ");
    while (n < 1)
    {
        n = get_int("Height: ");
    }

    for (int i = 1; i < n + 1; i++)
    {
        for (int j = 1; j < n - i + 1; j++)
        {
            printf(" ");
        }
        for (int k = 1; k < i + 1; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = 1; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week_6_assignment/sentimental-mario-less/mario.py
from cs50 import get_int
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(1, height+1):
    for k in range(1, height-i+1):
        print(" ", end="")
    for j in range(1, i+1):
        print("#", end="")
    print()
